src/index.ts

```diff
@@ -40,35 +40,43 @@ export function getCeloRTDBMetadata(environment: Environment): RTDBMetadata[] {
   ]
 }
 
-export function _getTokensInfo(
-  environment: Environment,
-): Record<Network, TokenInfo[]> {
-  return environment === 'mainnet'
-    ? {
-        [Network.ethereum]: addTokenIds(
-          EthereumMainnetTokensInfo,
-          NetworkId['ethereum-mainnet'],
-        ),
-        [Network.celo]: addTokenIds(
-          CeloMainnetTokensInfo,
-          NetworkId['celo-mainnet'],
-        ),
-      }
-    : {
-        [Network.ethereum]: addTokenIds(
-          EthereumTestnetTokensInfo,
-          NetworkId['ethereum-sepolia'],
-        ),
-        [Network.celo]: addTokenIds(
-          CeloTestnetTokensInfo,
-          NetworkId['celo-alfajores'],
-        ),
-      }
+const networkIdToTokensInfo: Record<NetworkId, Omit<TokenInfo, 'network'>[]> = {
+  [NetworkId['celo-mainnet']]: CeloMainnetTokensInfo,
+  [NetworkId['celo-alfajores']]: CeloAlfajoresTokensInfo,
+  [NetworkId['ethereum-mainnet']]: EthereumMainnetTokensInfo,
+  [NetworkId['ethereum-sepolia']]: EthereumSepoliaTokensInfo,
+}
+
+const networkIdToNetwork: Record<NetworkId, Network> = {
+  [NetworkId['celo-mainnet']]: Network.celo,
+  [NetworkId['celo-alfajores']]: Network.celo,
+  [NetworkId['ethereum-mainnet']]: Network.ethereum,
+  [NetworkId['ethereum-sepolia']]: Network.ethereum,
+}
+
+export function getTokenId(
+  { isNative, address }: Partial<TokenInfo>,
+  networkId: NetworkId,
+): string {
+  return `${networkId}:${isNative ? 'native' : address}`
+}
+
+export function _getTokensInfo(networkIds: NetworkId[]): {
+  [tokenId: string]: TokenInfo
+} {
+  const output: { [tokenId: string]: TokenInfo } = {}
```

Comment:
This flattens assets across all networks into a single mapping -- the other obvious option is to return a list of objects, where each object corresponds to a single network+env. FWIW, I like this option better. I can't see any reason why we'd care about separating them, since the identifiers are unique anyhow, although if some consumer explicitly wants separate lists for each network+env, it might be cumbersome. I don't think that'll ever be the case though, so prolly not something to worry about.

Reply:
this is simpler and seems to meet the need for now-- I suggest we try to stay nimble and switch to the other format you're suggesting if we see a need for it

Reply:
oh sorry, maybe i wasn't clear enough -- i prefer the option that you've gone with here